docs/failover-state-machine.rst

```diff
@@ -56,25 +56,36 @@ primary (goal: primary) and B secondary (goal: secondary).
 State reference
 ---------------
 
-For a graph of the states and their transitions, see
-:ref:`state_machine_diagram`.
+The following diagram shows the pg_auto_failover State Machine. It's missing
+links to the ``single`` state, which can always been reached when removing
+all the other nodes.
 
-**Init**
+.. figure:: ./tikz/fsm.svg
+   :alt: pg_auto_failover Finite State Machine diagram
```

Comment:
That's because we don' have such a transition?

```
$ pg_autoctl do fsm gv | grep -- '-> stop_replication'
    prepare_promotion -> stop_replication [ label = "Prevent against split-brain situations." ];
```

Reply:
For completeness:

```
pg_autoctl do fsm gv | grep -- 'stop_replication ->'
    stop_replication -> single [ label = "Went down to force the primary to time out, but then it was removed" ];
    stop_replication -> wait_primary [ label = "Confirmed promotion with the monitor" ];
```